cmake/onnxruntime_common.cmake

```diff
@@ -138,4 +138,10 @@ endif()
 
 if (onnxruntime_WINML_NAMESPACE_OVERRIDE STREQUAL "Windows")
   target_compile_definitions(onnxruntime_common PRIVATE "BUILD_INBOX=1")
-endif()
\ No newline at end of file
+endif()
+
+# check if we need to link against libatomic due to std::atomic usage by the threadpool code
+# e.g. Raspberry Pi requires this
+if (onnxruntime_LINK_LIBATOMIC)
+  target_link_libraries(onnxruntime_common atomic)
```

Comment:
Could you add it to onnxruntime_EXTERNAL_LIBRARIES  var instead? Like https://github.com/microsoft/onnxruntime/blob/master/cmake/CMakeLists.txt#L770

Otherwise, you need to specify a visibility keyword here. Like

target_link_libraries(onnxruntime_common PUBLIC atomic)